sos/plugins/ansible.py

```diff
@@ -0,0 +1,40 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class Ansible(Plugin, RedHatPlugin, UbuntuPlugin):
+    """Ansible configuration management
+    """
+
+    plugin_name = 'ansible'
+    profiles = ('system',)
+
+    packages = (
+        'ansible',
+        'ansible1.9'
+    )
+
+    def setup(self):
+        self.add_copy_spec(
+            "/etc/ansible/"
+        )
+
+        self.add_cmd_output([
+            "ansible all -m ping -vvvv",
```

Comment:
Can't the ping hang (in case of e.g. network issues or service hung)?

Maybe I am too afraid of, but I recall some such checks that were timeouted after 5 minutes in these situations. Dont know how this ping behaves.